### DISCUSSION TITLE
Problem with kata

### DISCUSSION THREAD

**Discussion by Extend777** [codewars/codewars.com#3075] [Q&A] [answered]
In general, I had the following problem. Once upon a time, when I was young and stupid, for a long time I dreamed of successfully completing the solution to the most difficult 1kyu kata. As you might expect, it didn't work out for me. And for some reason a very bad idea came into my head, to copy a ready-made kata solution from another person’s Github. Honestly, now I greatly regret this action.
As a result, the points and rank that were obtained by solving this kata were taken away from me, but the check mark that displayed the status of the completed task remained.
Now that I have gained experience and wanted to repeat this, but on my own without cheating, it seems to me that I will no longer receive points and rank. My question is, is it possible to somehow exclude this kata from my “solved” ones and repeat it as if it were being done for the first time?

**Comment by hobovsky** [comment, marked as answer]
No, it is not possible. Once you solved the kata with a copied solution, you got access to all solutions and you had a chance to get all knowledge how to solve it. There is no way to un-remember the solution and solve the kata again to get points.

For the same reason, the system does  not allow to remove the kata from your list of solved kata. You did unlock it, after all, its just that your points are gone.

Tjere is many, many more difficult, challenging, purple kata. You will easily find something else to do. Me myself, i have one or two purple kata which i forfeited with a missclick and will be never able to solve them for points.

**Reply by Extend777** [reply]
It's sad.. But thanks for answer :)
